File: patterns/c-patterns/pattern27.c
// Pattern 27: ...

// EDCBA
// *DCBA
// **CBA
// ***BA
// ****A

#include <stdio.h>
int main()
{
    int i, j;
    int n = 4;
    for (i = n; i >= 0; i--)
    {
        for (j = n; j >= 0; j--)
        {
            if (j > i)
                printf("*");
            else
                printf("%c", j + 65);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern27.js
// Pattern 27: ...

// EDCBA
// *DCBA
// **CBA
// ***BA
// ****A

const n = 4;

for (let i = n; i >= 0; i--) {
    let row = '';

    for (let j = n; j >= 0; j--) {
        if (j > i) {
            row += '*';
        } else {
            row += String.fromCharCode(j + 65);
        }
    }

    console.log(row);
}